packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py

```diff
@@ -54,7 +54,7 @@ async def _call(
             user_id=user_id,
             limit=limit,
             offset=offset,
-            timeout_s=20 * RPC_REQUEST_DEFAULT_TIMEOUT_S,
+            timeout_s=40 * RPC_REQUEST_DEFAULT_TIMEOUT_S,
```

Comment:
Q: I don't get why you have to multiply here the value, can't we just use one single constant, which is simpler to read and comprehend?

Reply:
I find that this way we have a good reference